docs/content/preview/faq/yugabyte-platform.md

```diff
@@ -123,3 +138,88 @@ Yes, you can control what YugabyteDB Anywhere is spinning up. For example:
+
+For on-prem manual provisioning, an SSH user does not have sudo access, but sudo is available on the local YugabyteDB node with a password.
+
+For more information, refer to [install node agent](../../yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual/#install-node-agent).
+
+### Why does YBA UI ask for SSH if the node agent is installed manually and can replace SSH?
+
+This is because of the legacy UI. There are some cases where an SSH key is needed for integrity but a fictitious non-working SSH key can be supplied.
+
+### What is registration and unregistration in node agent?
+
+Node agent is a secure service that performs authentication on every remote call from YBA. The service requires a set-up to make:
+
+- YBA aware of the node agent service.
+- the node agent trust YBA and vice-versa.
+This step is called registration. Note that registration has nothing to do with a provider.
+
+Unregistration is the step to disconnect YBA and a node agent from each other.
+
+### Why does node agent installation ask for provider and other details during on-prem manual node agent setup?
+
+Node agent registration internally does not need configuration values such as region, availability zone, and so on, but it still requests the details because of the additional preflight check capability that it provides. It stores the configuration details in a config file called `$NODE_AGENT_HOME/config/config.yml`, and the values are used when the preflight check is run, or the node is added to the node instance of the provider. This is convenient when you run the command as you don't have to provide the input values again.
+
+### What happens if I want to change the provider settings of a node agent, or move it to another provider?
+
+A node agent does not need to be unregistered to move a provider as it is not tied to a provider. Following are the steps to change the provider prior to YBA version 2.18.5:
+
+1. Remove the node instance from the provider if it is already added.
+1. Unregister the existing node agent (This step is only required prior to YBA v2.18.5).
+1. Stop the `systemd` service.
+1. Run the installation again, or the command `node-agent node configure`. Running the command is faster as it does not install node agent again.
+1. Start the `systemd` service.
+
+For YBA version 2.18.5 and later, do the following:
+
+1. Remove the node instance from the provider if it is already added using the command `node-agent node delete-instance`.
+1. Run the command `node-agent node configure`.
+
+As long as the IP does not change, the node agent does not try to register again.
+
+Note that the first step to change the provider is very important because after the provider configuration information changes in the config file as part of the second step, the node agent can't find the node to delete it as the scope is always with the provider and availability zone.
```

Comment:
first step to change the provider  => first step to remove the node instance from the provider